karma.conf.js

```diff
@@ -33,6 +33,8 @@ export default function configKarma(karma: Object) {
       ...karmaConfig.files,
     ],
 
+    exclude: ["test/globals.js"],
```

Comment:
from this [comment](https://github.com/paypal/paypal-common-components/pull/95/files#r1581586041)